physionet-django/user/managers.py

```diff
@@ -1,32 +1,71 @@
-from django.db.models import DateTimeField, ExpressionWrapper, QuerySet, F, Q
+from django.db.models import (DateTimeField, ExpressionWrapper, QuerySet, F, Q,
+                              OuterRef, Subquery, Case, When)
 from django.utils import timezone
 
-from user.enums import TrainingStatus
-
+from user.enums import TrainingStatus, RequiredField
+from training.models import Course
 
 class TrainingQuerySet(QuerySet):
     def get_review(self):
-        return self.filter(status=TrainingStatus.REVIEW)
+        """
+        Get the document-based or URL-based training objects for the user, that are in the status REVIEW.
+        """
+        return self.filter(
+            Q(status=TrainingStatus.REVIEW),
+            Q(training_type__required_field=RequiredField.DOCUMENT)
+            | Q(training_type__required_field=RequiredField.URL)
+        )
+
+    def in_progress(self):
+        """
+        Get the on-platform training objects for the user, that are in the status REVIEW.
+        """
+        return self.filter(
```

Comment:
it's odd that we are using the `TrainingStatus.REVIEW` here. Is there a reason why we don't introduce a new `TrainingStatus.IN_PROGRESS` status?

Reply:
Training has other mechanisms linked to it, and hence I did not want to introduce a new type of training status, which might introduce new bugs, given I might miss handling it somewhere. 

Furthermore, the REVIEW & the IN_PROGRESS are practically the same status, given one is for on-platform courses and other for document-submission based courses. Hence, to make sure we don't introduce unnecessary bugs accidentally, I went ahead with REVIEW.

Reply:
As discussed in the call with @tompollard & @bemoody, I've implemented a new `IN_PROGRESS` status, which is now used by on-platform trainings.